tests/calculator-homework.spec.js

```diff
@@ -0,0 +1,100 @@
+const { test, expect } = require('@playwright/test');
+const {calculatorStartPage} = require('../pages/calculatorStartPage')
+
+test.describe('', () => {
+  let page;
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    startPage = new calculatorStartPage(page);
+  });
+  test.beforeEach(async () => {
+    await startPage.goto();
+  });
+
+  const builds = ['0','1','2','3','4','5','6','7','8','9'];
+
+
+
+builds.forEach(build => {
+test.only(`add function is correct for ${build}`, async () => {
+  await startPage.goto();
+  await page.selectOption('#selectBuild', build);
+  await page.fill('#number1Field', '1');
+  await page.fill('#number2Field', '1');
+  await page.selectOption('#selectOperationDropdown', '0');
+  await page.click('#calculateButton');
+  const result1 = await page.inputValue('#numberAnswerField');
```

Comment:
const result would be enough. Number at the end is not needed (applies to the variables below too)